src/Native/System.Native/pal_networking.h

```diff
@@ -84,3 +84,24 @@ extern "C" int32_t GetNameInfo(const uint8_t* address,
                                int32_t flags);
 
 extern "C" int32_t GetHostName(uint8_t* name, int32_t nameLength);
+
+struct LinkLayerAddressInfo
+{
+    uint32_t InterfaceIndex; // The index of the interface to which this address belongs.
+    uint8_t AddressBytes[8]; // A pointer to the bytes containing the address.
+    uint8_t NumAddressBytes; // The number of bytes actually stored in the address.
+    uint8_t __padding;
```

Comment:
Ah yeah, I think Pat's change removes the reserved bits and lets the compiler take care of it. If not, I'll fix that up since it might not be correct for a specified platform